python_modules/libraries/dagster-sigma/setup.py

```diff
@@ -35,7 +35,7 @@ def get_version() -> str:
         "Operating System :: OS Independent",
     ],
     packages=find_packages(exclude=["dagster_sigma_tests*"]),
-    install_requires=[f"dagster{pin}", "sqlglot", "aiohttp"],
+    install_requires=[f"dagster{pin}", "sqlglot", "aiohttp<3.11"],
```

Comment:
is this a test only restriction? pin the specific version only in the test extra?

Reply:
good idea